pinot-query-runtime/src/test/resources/queries/WindowFunctions.json

```diff
@@ -51,6 +51,50 @@
           [768]
         ]
       },
+      {
+        "description": "Single empty OVER() count(*) without select col",
+        "sql": "SELECT COUNT(*) OVER() FROM {tbl}",
+        "outputs": [
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16],
+          [16]
+        ]
+      },
+      {
+        "description": "Single empty OVER() count(*) without select col",
+        "sql": "SELECT 42, COUNT(*) OVER() FROM {tbl}",
```

Comment:
Same question here

Reply:
Yes this works in PostgreSQL:

```
dvdrental=# select 42, COUNT(*) OVER() from payment;
 ?column? | count 
----------+-------
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
       42 | 14596
```

There are many types of custom window frames that works in PostgreSQL but isn't allowed by Apache Calcite. I'll work on a document to list out the ones I know about. I assume that if we want to move towards PostgreSQL compliance, we want to allow these kinds of queries, right?

Reply:
Yep that's right. Let's get that document ready.